### PR TITLE
PR for Models and Relationships

### PR DESCRIPTION
**Pull request by shechframgia**
I added new model named Article with relationships in User model :D

### CODE REVIEW

File: app/Article.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    public function user()
+	{
+		return $this->belongsTo('App\User');
```
Comment (karlpandacan) [inline app/Article.php]: use "User::class"

File: app/User.php [hunk 2]
```diff
@@ -23,4 +23,9 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    public function articles()
+    {
+        return $this->hasMany('App\Article');
```
Comment (karlpandacan) [inline app/User.php]: Same with here https://github.com/jsnulla/blog_1/pull/2/files?diff=split#r54524743

**Comment by karlpandacan** [comment]
Move your Models to App\Models\

File: app/Models/Article.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    public function user()
+	{
+		return $this->belongsTo(User::class);
```
Comment (karlpandacan) [inline app/Models/Article.php]: please check the indention.

File: app/Models/User.php [hunk 4]
```diff
@@ -23,4 +23,9 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
```
Comment (jsnulla) [inline app/Models/User.php]: Please define `$fillables` as well.

**Comment by karlpandacan** [comment]
No Migration Files

**Comment by jsnulla** [comment]
OK :+1:

**Pull request merged**